src/lib/components/Creatibutors/CreatibutorsModal.js

```diff
@@ -335,7 +337,7 @@ export class CreatibutorsModal extends Component {
         onSubmit={this.onSubmit}
         enableReinitialize
         validationSchema={this.CreatorSchema}
-        validateOnChange={false}
+        validateOnChange={true}
```

Comment:
I have done a few tests and only with this change (without the `attemptedSumbit` fix) it looks like working.
Would you have some time to check this together?

Reply:
Yes, it works that way. The `attemptedSubmit` is just there to prevent the error messages to pop up when moving on to the next field :)